ports/lhasa/portfile.cmake

```
@@ -0,0 +1,42 @@
+set(LHASA_VERSION 0.4.0)
+vcpkg_download_distfile(ARCHIVE
+    URLS "https://github.com/fragglet/lhasa/releases/download/v0.4.0/lhasa-${LHASA_VERSION}.tar.gz"
+    FILENAME "lhasa-${LHASA_VERSION}.tar.gz"
```

```suggestion
vcpkg_download_distfile(ARCHIVE
    URLS "https://github.com/fragglet/lhasa/releases/download/v0.4.0/lhasa-${VERSION}.tar.gz"
    FILENAME "lhasa-${VERSION}.tar.gz"
```
You can use `${VERSION}` directly, it reads the "version" field in vcpkg.json.